lavamoat/browserify/beta/policy.json

```diff
@@ -565,6 +562,7 @@
         "addEventListener": true,
         "atob": true,
         "btoa": true,
+        "clearTimeout": true,
```

Comment:
The messaging library has added a clearTimeout in a recent patch update [here](https://github.com/firebase/firebase-js-sdk/blob/main/packages/messaging/CHANGELOG.md#01215)